subprojects/core/src/main/java/org/gradle/execution/taskgraph/TaskPlanExecutor.java

```diff
@@ -20,7 +20,7 @@
 import org.gradle.api.internal.TaskInternal;
 
 /**
- * Will be merged with {@link org.gradle.internal.operations.BuildOperationProcessor}
+ * Will be merged with {@link org.gradle.internal.operations.BuildOperationExecutor}
```

Comment:
I agree in general. But I think this is unrelated to this PR. Should we add an issue of some sort?